.eslintrc.cjs

```diff
@@ -18,10 +18,7 @@ const config = {
   overrides: [
     {
       files: ['.eslintrc.*', '.prettierrc.*', '*.config.js'],
-      rules: {
-        '@typescript-eslint/no-var-requires': 'off',
-        'no-undef': 'off',
-      },
```

Comment:
Without this line, there will be something like this. Are there better option to this?

![Screenshot 2566-05-17 at 01 23 07](https://github.com/creatorsgarten/kaogeek-discord-bot/assets/59821765/a160ecc5-d753-41a0-b4b5-b25b25a5940e)

Reply:
Hmm, it's not error in my editor tho. We can just disable for only this line. I would not recommend disable a rule for only this purpose.

Reply:
It only disable the rule for eslintrc and prettierrc file. That should work?

Reply:
It should, but I cannot surely confirm as I am not encountering this error. Would you mind trying it out? Just `// eslint-disable-next-line no-undef` should work, if the error is from eslint.

Reply:
I updated the eslintrc to use { env: { node: true } }
Should follow best practice now :+1: